src/extension.ts

```diff
@@ -164,6 +175,15 @@ export async function activateLanguageServer(context: vscode.ExtensionContext, s
 		const params: ExecuteCommandParams = { command: "ntt.test", arguments: [args] };
 		await client.sendRequest(ExecuteCommandRequest.type, params);
 	}));
+	context.subscriptions.push(
+		vscode.workspace.onDidChangeConfiguration((e: vscode.ConfigurationChangeEvent) => {
+			if (!e.affectsConfiguration('ttcn3')) {
```

Comment:
Are we required to ignore non-ttcn3 configuration changes? I would imagine that changes in CMake configuration could be interesting for our language server.

Reply:
I removed the constraint